charts/argo-cd/Chart.yaml

```diff
@@ -26,6 +26,8 @@ annotations:
     fingerprint: 2B8F22F57260EFA67BE1C5824B11F800CD9D2252
     url: https://argoproj.github.io/argo-helm/pgp_keys.asc
   artifacthub.io/changes: |
+    - kind: changed
+      description: Repository credentials Secret will use stringData to increase compatibility with plugins like helm-secrets
     - kind: removed
```

Comment:
You need to remove the old stuff please.

Reply:
thanks @tico24 for the review. What do you mean by the old stuff. This PR only changes something that's there. I'm not sure if I understod this changelog entirely

Reply:
You can look at other prs for examples. The changelog should only contain your change. Remove the old stuff.

Reply:
He was referring to old changelog entries, only the last one needs bo he in the Pr as this is automatically constructed as part of release.